src/api/ateliereLive/pipelines/pipelines.ts

```diff
@@ -285,30 +270,27 @@ export async function connectControlPanelToPipeline(
   await Promise.all(connectSendersToReceivers);
 }
 
-function buildPipelineSettingsBody(
-  pipeline: PipelineSettings
+function buildOutputStreamSettings(
+  output: PipelineOutput
 ): PipelineOutputSettings[] {
-  const outputSettings = pipeline.program_output.map((output) => {
+  const outputSettings = output.streams.map((s) => {
     return {
-      audio_format: output.audio_format,
-      audio_kilobit_rate: output.audio_kilobit_rate,
-      format: output.format,
-      local_ip: output.local_ip,
-      local_port: output.srt_mode === 'caller' ? 0 : output.port,
```

Comment:
Not completely following why this is set to 0 if mode is caller instead of s.local_port, from API:
"The port to bind for the local interface. In listener mode this is the port listening for incoming SRT connections, for caller mode this is the local port used in the connection.
If set to 0, the Ingest will select a random port."